File: app/src/main/java/br/org/fundatec/heroes_app/home/view/SplashActivity.java
package br.org.fundatec.heroes_app.home.view;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import br.org.fundatec.heroes_app.R;

public class SplashActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                mostrarMainActivity();
            }
        }, 3000);
    }

    private void mostrarMainActivity() {
        Intent intent = new Intent(
                SplashActivity.this,HomeActivity.class
        );
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/br/org/fundatec/heroes_app/home/view/SplashActivity.kt
package br.org.fundatec.heroes_app.home.view

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import br.org.fundatec.heroes_app.R

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        getSupportActionBar()?.hide()
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        val handle = Handler()
        handle.postDelayed({ chamarLoginActivity() }, 3000)

    }

    private fun chamarLoginActivity() {
        val intent = Intent(this@SplashActivity, LoginActivity::class.java)
        startActivity(intent)
        finish()
    }

}


